WooCommerce/Classes/ViewRelated/Products/Edit Product/Downloads/Downloadable FIle List/ProductDownloadDnD.swift

```diff
@@ -0,0 +1,75 @@
+import Foundation
+import CoreServices
+import Yosemite
+
+/// A wrapper around ProductDownload, to make it compatible in using as Drag and Drop data source in an table view. Represents a ProductDownload entity.
+///
+public class ProductDownloadDnD: NSObject, Codable {
+    public let download: ProductDownload
+
+    /// ProductDownloadDnD initializer.
```

Comment:
I think it's clear that we are referring to itself.
```suggestion
    ///Initializer.
```

Reply:
😄